CHANGES.md

```diff
@@ -1,3 +1,19 @@
+## edge-19.4.1
+
+* CLI
+  * Introduced an `upgrade` command! This allows an existing Linkerd Control Plane to be reinstalled or reconfigured
+  * The `--proxy-auto-inject` flag  from the `install` command has been completely revamped
```

Comment:
Is this making reference to #2562? If so, this means the CP remembers that flag, which I believe is implicit in the previous bullet point... If not, could you please elaborate a bit more what this change is about?

Reply:
Yeah, I was wondering if this bullet point is necessary, as it doesn't change the functionality from a user's POV.

Reply:
@alpeb Yes I was referencing that and the encompassing auto-inject changes. It was significant work, but not user facing so I will remove it.